src/components/SearchForm.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+function SearchForm({ onInputChange, language }) {
+  return (
+    <Form aria-labelledby="formHeader" autoComplete="off">
+      {language === "en" ? (
+        <h1 id="formHeader">What do you want to buy?</h1>
+      ) : (
+        <h1 id="formHeader">Was willst du kaufen?</h1>
```

Comment:
I would recommend to use `label`s here instead of headings. And then connect them to the input element.